core/src/com/unciv/ui/newgamescreen/GameOptionsTable.kt

```diff
@@ -122,7 +122,7 @@ class GameOptionsTable(
         if (maxCityStates == 0) return null
 
         add("{Number of City-States}:".toLabel()).left().expandX()
-        val slider = UncivSlider(0f, maxCityStates.toFloat(), 1f) {
+        val slider = UncivSlider(0f, maxCityStates.toFloat(), 1f, initial = 6f) {
```

Comment:
See line 131 (I've been here, I think I converted these to the new widget, why didn't I see that :see_no_evil: )